src/components/SlideButton/SlideButton.scss

```diff
@@ -0,0 +1,74 @@
+@import "../../styles/colors";
+@import "../../styles/typography";
+
+.wrapper {
+  background-color: $white;
+  padding: 16px;
+  width: 100%;
+  box-sizing: border-box;
+}
+.slider-wrapper {
```

Comment:
(nit) please add a single empty line between each selector

Reply:
Please also make this change ^

Reply:
This one is done!!